L1Trigger/L1TZDC/python/l1tZDCEtSums_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from L1Trigger.L1TZDC.l1tZDCProducer_cfi import l1tZDCProducer as _l1tZDCProducer
```

Comment:
@missirol I am working through a separate packer problem right now, but running into some ZDC things I don't understand and I'm trying to chase it down to the base of the ZDC emulation, which you must recently touched in this PR. I'm confused by this import (and how I'm not getting python problems) because `L1Trigger.L1TZDC.l1tZDCProducer_cfi` doesn't seem to exist. A quick search on github doesn't seem to reveal any mention of this file other than right here, and I can't find it under L1Trigger/L1TZDC. 

This PR removed what I remembered as the previous L1Trigger ZDC emulation config `etSumZdcProducer_cfi`, was this perhaps supposed to be a rename of that configuration that didn't end up in the PR? Am I missing some other thing that was done to the ZDC emulation to make this import and producer work?